config/manager/manager.yaml

```diff
@@ -70,6 +70,9 @@ spec:
         - /manager
         args:
         - --leader-elect
+        env:
+          - name: CLUSTER_NAME
+            value: CLUSTER_NAME_VALUE
```

Comment:
maybe we should add a comment here to replace it with one's actual cluster name?